packages/hash/frontend/src/blocks/page/ComponentView.tsx

```diff
@@ -273,6 +276,16 @@ export class ComponentView implements NodeView<Schema> {
         }),
       );
     }
+
+    const isParagraph = this.node.type.name === paragraphBlockComponentId;
```

Comment:
We now have two ways to detect the paragraph block – we should be consistent. https://github.com/hashintel/hash/blob/d5f1d742af4d5ad88012ab2a2d2d3753393bc143/packages/hash/frontend/src/blocks/page/createEditorView.ts#L153 

Also, the fact that `node.type.name` is the componentId should be relied on. It's an implementation detail. That could change quite easily. If you *need* to get the component id from the name, you should use `componentNodeToId` so we can track them, but really you should use `this.block.meta.componentId` which is more explicit about what you're trying to do.

Reply:
Updated here https://github.com/hashintel/hash/pull/953/commits/5bebff617f57205a74e1f4ccba39a2e062624377